src/Slider/index.scss

```diff
@@ -0,0 +1,39 @@
+.nds-slider {
+  position: relative;
+  width: 300px;
+  touch-action: none;
+
+  .rail {
+    position: absolute;
+    background-color: rgba(var(--theme-rgb-primary), var(--alpha-20));
+    height: 2px;
+    width: 100%;
+  }
+
+  .rail-active {
+    position: relative;
+    background-color: var(--theme-primary);
+    height: 2px;
+  }
+
+  .track {
+    position: relative;
+    height: 15px;
+    width: 100%;
+    display: flex;
+    align-items: center;
+  }
+
+  .thumb {
+    top: 8px;
+    width: 16px;
+    height: 16px;
+    border-radius: 50%;
+    box-sizing: border-box;
+    background-color: var(--theme-primary);
+  }
+
+  .output-container {
+    margin-bottom: var(--space-l);
+  }
```

Comment:
so yeah this was another thing i should have brought up in the PR. 

The text displayed in this `<output>` is what's adjacent to the slider in the designs (designs below)

![image](https://github.com/narmi/design_system/assets/4285085/0b30df18-14a7-4069-953a-9a7c8be3166b)

it's not a label, since it's describing the values of the input, not what the input is for. For that reason, I've also included a visually hidden `<label>`, which serves to provide an accessible label for the input. This might be confusing to a consumer of this NDS component though. 

But to answer this question - this is styled as the designs, at least for the only usage of this component that i've seen. So maybe keep it as is? 

open to thoughts about the output/label though.

Reply:
Makes sense to me! I think using `output` is a nice choice here